### ISSUE TITLE
feature: Show active virtual branches in console

### ISSUE THREAD

**Issue by blowery** [gitbutlerapp/gitbutler#2890]
Lots of folks have fancy terminals that show the currently active branch. But when gitbutler is running things, that's always just `gitbutler/integration`. It might be nice to have a cli tool that exposes the names of the currently applied virtual branches? 

This feels like part of a bigger issue around how this should feel for folks who live on the command line. gitbutler also breaks things like changed lines, how far you're ahead of or behind a tracking branch, etc etc.

**Comment by mtsgrd** [comment]
We are definitely interested in this, but I'm unsure what the timeline could be. There are quite a few improvements we want to make to the rust codebase before we can reliably separate out some CLI functionality.

Fyi @krlvi @Qix-

**Comment by krlvi** [comment]
If i understand correctly, this is about showing the refs of vbranches in `git branch` listing. Currently the vbranches can be found in `ls -l .git/refs/gitbutler`.
One reason we dont show them as regular branches (written in `.git/refs/heads`) is to not pollute the name space. I think another reason is the state management can get weird if those refs updated manually outside of gb. 

As @mtsgrd mentioned, we'd love to add a gitbutler cli that exposes functionality that way too
